spec/factories/users_organizations.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :users_organization do
+    user { nil }
```

Comment:
just for my interest .. why is this { nil } here?

Reply:
this was auto generated when I did

```
be rails g model UsersOrganizations user:references organization:references
```

we could maybe remove the factory as it is unused.